R/pkg/R/context.R

```diff
@@ -345,7 +351,7 @@ spark.getSparkFilesRootDirectory <- function() {
 #'}
 #' @note spark.getSparkFiles since 2.1.0
 spark.getSparkFiles <- function(fileName) {
-  callJStatic("org.apache.spark.SparkFiles", "get", as.character(fileName))
+  file.path(spark.getSparkFilesRootDirectory(), as.character(fileName))
```

Comment:
or that fileName can be something file.path doesn't handle but SparkFiles.get() can? like an absolute path?